cli/src/cmd_auth.rs

```diff
@@ -440,126 +443,112 @@ impl CmdAuthLogout {
-
-        //     let client = ctx.api_client(hostname)?;
-
-        //     let mut host_status: Vec<String> = vec![];
-
-        //     match client.hidden().session_me().await {
-        //         Ok(session) => {
-        //             // TODO: this should be the users email or something consistent with login
-        //             // and logout.
-        //             let email = session.id.to_string();
-        //             // Let the user know if their token is invalid.
-        //             /*if !session.is_valid() {
-        //             host_status.push(format!(
-        //                 "{} Logged in to {} as {} ({}) with an invalid token",
-        //                 cs.failure_icon(),
-        //                 hostname,
-        //                 cs.bold(&email),
-        //                 token_source
-        //             ));
-        //             failed = true;
-        //             continue;
-        //             }*/
-        //             host_status.push(format!(
-        //                 "{} Logged in to {} as {} ({})",
-        //                 cs.success_icon(),
-        //                 hostname,
-        //                 cs.bold(&email),
-        //                 token_source
-        //             ));
-        //             let mut token_display = "*******************".to_string();
-        //             if self.show_token {
-        //                 token_display = token.to_string();
-        //             }
-        //             host_status.push(format!("{} Token: {}", cs.success_icon(), token_display));
-        //         }
-        //         Err(err) => {
-        //             host_status.push(format!(
-        //                 "{} {}: api call failed: {}",
-        //                 cs.failure_icon(),
-        //                 hostname,
-        //                 err
-        //             ));
-        //             failed = true;
-        //             continue;
-        //         }
-        //     }
+            // TODO: this should be the users email or something consistent with login
+            // and logout.
+            let email = user.id.to_string();
 
-        //     status_info.insert(hostname.to_string(), host_status);
-        // }
+            // TODO: Once tokens have expiry dates, let them know if their token is invalid.
 
-        // if !hostname_found {
-        //     writeln!(
-        //         ctx.io.err_out,
-        //         "Hostname {} not found among authenticated Oxide hosts",
-        //         self.host.as_ref().unwrap().as_str(),
-        //     )?;
-        //     return Err(anyhow!(""));
-        // }
+            host_status.push(format!("Logged in to {} as {}", host, &email,));
+            let mut token_display = "*******************".to_string();
+            if self.show_token {
+                token_display = info.token.to_string();
+            }
```

Comment:
```suggestion
            let token_display = 
            if self.show_token {
                info.token.to_string();
            } else {
            "*******************".to_string()
            };
```

^ I think something like this would be more idiomatic.

Reply:
Ha! TIL you can assign if/else results to a variable like that. Nice!